### DISCUSSION TITLE
Is it possible to force disable and lsp?

### DISCUSSION THREAD

**Discussion by ncpa0cpl** [zed-industries/zed#14393] [Feature Requests]
I wanted to completely disable ESLint lsp, but I couldn't find anything in the docs on how to do something like that. Is it possible to shut if off?

**Comment by SomeoneToIgnore** [comment]
It looks like you can do something like 

https://github.com/zed-industries/zed/blob/89fbd6528f688cc98eaa4b3002f4b41da6c7b520/assets/settings/default.json#L742-L744

per language, in the `"languages"` section of you settings, using `!eslint` syntax.

**Reply by ncpa0cpl** [reply]
thank you
